python/lsst/pipe/tasks/dcrAssembleCoadd.py

```diff
@@ -60,41 +60,46 @@ class DcrAssembleCoaddConfig(CompareWarpAssembleCoaddConfig):
             "If not set, skips calculating convergence and runs for ``maxNumIter`` iterations",
         default=True,
     )
+    useProgressiveGain = pexConfig.Field(
+        dtype=bool,
+        doc="Allow the relative weight of the new model solution to increase with each iteration?",
```

Comment:
This sentence fragment does not make sense to end with a question mark. Either make it a proper statement or reword it to a proper question. I think our standard says the docs should be statements.